distsql/select_result.go

```diff
@@ -448,8 +448,10 @@ func (s *selectResultRuntimeStats) String() string {
 			buf.WriteString(", rpc_time: ")
 			buf.WriteString(time.Duration(copRPC.Consume).String())
 		}
-		buf.WriteString(fmt.Sprintf(", copr_cache_hit_ratio: %v",
-			strconv.FormatFloat(float64(s.CoprCacheHitNum)/float64(len(s.copRespTime)), 'f', 2, 64)))
+		if s.CoprCacheHitNum > 0 {
```

Comment:
Maybe it's better to preserve hit ratio 0 when cache is actually enabled

Reply:
I think if the value is 0, then it can be ignored. If the cache is enabled and take effect, the value will not be 0.

@SunRunAway What do you think?

Reply:
> Explicit is better than implicit.

I think it's better to keep zero if copr-cache is enabled..